### DISCUSSION TITLE
Docs on how to deploy to cloudflare

### DISCUSSION THREAD

**Discussion by shuntl** [wasp-lang/wasp#1987] [Archived]
In the title - would love some docs on how to deploy to more places. Evaluating whether I go with digital ocean droplet or cloudflare, etc.

**Comment by Martinsos** [comment]
Thanks @shuntl , we would love to add those at some point!
In the meantime, if it does happen that you do deploy to cloudflare, any info you share about your experience and steps you took can help in creating those docs. PRs are also welcome!
